recipe/meta.yaml

```diff
@@ -9,14 +9,12 @@ package:
 
 source:
   url: https://download.gnome.org/sources/{{ name }}/{{ '.'.join(version.split('.')[:2]) }}/{{ name }}-{{ version }}.tar.xz
-  sha256: 80beae6728c134521926affff9b2e97125749b38d38744dc901f4010ee3e7fa7
+  sha256: 3d66ea3aec7c3f8f1f83d89db3e78e18e36adc22b8bba45298119f0b3ea50060
   patches:
-    - meson-rpaths.patch  # [linux]
-    - gcc7-rpath-link.patch  # [linux]
```

Comment:
Did these get upstream or they are not applying cleanly and need a refresh?

Reply:
I don't know to be honest. But it appears that they are no longer necessary as the tests run fine without them.

Reply:
Tests may be misleading, they may not cover that part. I'll defer to @pkgw to merge this one b/c I don't really know enough about `gobject-introspection` to do it myself.

Reply:
The meson-rpaths patch should be unnecessary now that we're patching Meson (sigh). I'd be less sure about `gcc7-rpath-link`, but the comment in the patch explicitly says that building pygobject tests whether it works, and we're doing that now, so I think that issue must have been fixed upstream.